src/PageContent/Settings/AccountManager.js

```diff
@@ -0,0 +1,191 @@
+import React, { Component } from 'react';
+import { AccountType } from '../../Enums';
+import { StringFormat } from '../../Enums';
+
+class AccountManager extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            isEditing: false
+        };
+    }
+
+    render() {
+
+        return (
+            <div className="scs-0">
+                <div className="scs-0-content scs-1-content">
+                    <h5>Account Manager Details</h5>
+
+                    {!this.state.isEditing ? 
+                        
+                        <div>
+                           
+                            <div className="amd-row">
+                                <div className="amd-details amd-details-1">
+                                    <div className="amd-details-child">
+                                        <h6>{this.props.account.name}</h6>
+                                        <h6>{this.props.account.position}</h6>
+                                        <h6>{this.props.account.email}</h6>
+                                        <h6>{this.props.account.phone}</h6>
+                                    </div>
+                                    
+                                    <div className="amd-details-child">
+                                        <h6>Recieve Notifications via</h6>
+                                        <h6>Feature coming soon!</h6>
+                                        {/* {this.props.account.smsNotif ? <h6>SMS/Text Message</h6> : <div/>}
+                                        {this.props.account.emailNotif ? <h6>Email</h6> : <div/>} */}
+                                    </div>
+                                </div>
+                            </div>
+
+                            {this.props.secondaryContact ?
```

Comment:
`this.props.account.secondaryContact` same with all the below

Reply:
Fixed.